src/plugins/intel_gpu/src/kernel_selector/common/tensor_type.h

```diff
@@ -287,6 +287,31 @@ inline bool SimpleLayout(DataLayout l) {
     }
 }
 
+inline bool DoubleBlockedLayout(DataLayout l) {
+    switch (l) {
+        case DataLayout::bs_fs_yx_bsv16_fsv16:
+        case DataLayout::bs_fs_zyx_bsv16_fsv32:
+        case DataLayout::bs_fs_zyx_bsv16_fsv16:
+        case DataLayout::bs_fs_yx_bsv4_fsv4:
+        case DataLayout::bs_fs_yx_bsv8_fsv4:
+        case DataLayout::bs_fs_yx_bsv8_fsv2:
+        case DataLayout::bs_fs_zyx_bsv8_fsv4:
+        case DataLayout::bs_fs_zyx_bsv8_fsv2:
+        case DataLayout::bs_fs_yx_bsv16_fsv4:
+        case DataLayout::bs_fs_zyx_bsv16_fsv4:
+        case DataLayout::bs_fs_yx_bsv16_fsv2:
+        case DataLayout::bs_fs_zyx_bsv16_fsv2:
+        case DataLayout::bs_fs_yx_bsv4_fsv2:
+        case DataLayout::bs_fs_yx_bsv32_fsv32:
+        case DataLayout::bs_fs_yx_bsv32_fsv16:
+        case DataLayout::bs_fs_zyx_bsv32_fsv32:
+        case DataLayout::bs_fs_zyx_bsv32_fsv16:
+            return true;
+        default:
+            return false;
+    }
+}
+
```

Comment:
I'm afraid this might be a duplication of code, making it difficult to maintain. Any suggestion @vladimir-paramuzov ?

Reply:
@jade-cho  @isanghao what about checking of `block_sizes` field?

Reply:
Thanks for the comment. That is the field that I was looking for :)

Reply:
@sshlyapn @isanghao  Do you mean cldnn::format::block_sizes()? jitter uses DataTensor. It doesn't have any field and function to get block size of tensor. Please let me know where it is similar.

Reply:
Oh, sorry, I missed exact place where you need to use it...
I found the similar hardcoded list in CheckInputsOutputNoPitchSameDims function. So, I believe we should consider to impellent more generic solution. But as for the fix I think we can keep it as is
@jade-cho  @isanghao  what do you think?

Reply:
OK, let's first merge it though it doesn't look great :(